### ISSUE TITLE
Bug bounty/responsible disclosure framework

### ISSUE THREAD

**Issue by lrettig** [spacemeshos/pm#235]
Self-explanatory. We need to specify reporting rules and requirements/terms and conditions, what's in and out of scope, SLA for response, and bounties.

**Comment by lrettig** [comment]
Initial version of this is now up at https://github.com/spacemeshos/go-spacemesh/blob/develop/SECURITY.md. I applied today for a community account at Hackerone, which we'll use to manage bounties.

**Comment by lrettig** [comment]
Waiting to finalize information internally here on the size of the bounties. Have a call scheduled with HackerOne to discuss next steps.

**Comment by lrettig** [comment]
Had the call with HackerOne. Still waiting on internal approval and budgeting.

**Comment by lrettig** [comment]
Have a follow up call scheduled for tomorrow.
